helm_deploy/manage-recalls-api/values.yaml

```diff
@@ -25,7 +25,10 @@ generic-service:
     SPRING_PROFILES_ACTIVE: "logstash"
     APPLICATIONINSIGHTS_CONNECTION_STRING: "InstrumentationKey=$(APPINSIGHTS_INSTRUMENTATIONKEY)"
     GOTENBERG_ENDPOINT_URL: http://manage-recalls-api-gotenberg
-    SENTRY_IGNORED_EXCEPTIONS_FOR_TYPE: org.springframework.security.web.firewall.RequestRejectedException,uk.gov.justice.digital.hmpps.managerecallsapi.service.VirusFoundException
+    SENTRY_IGNORED_EXCEPTIONS_FOR_TYPE:
```

Comment:
That format works?

Reply:
@dazoakley As YAML guru does ^^^ change I've made look valid to you please?  My IDE is telling me it is correct as is various googling, i.e. this is a valid YAML Sequence which evaluates to the same object as an Array?  Ta.

Reply:
(Force pushed) latest commit to switch this to a single line per chat with @dazoakley : https://mojdt.slack.com/archives/C02ET0U1G4T/p1641899860007400?thread_ts=1641871219.002700&cid=C02ET0U1G4T